cluster-autoscaler/apis/provisioningrequest/autoscaling.x-k8s.io/v1beta1/types.go

```diff
@@ -201,3 +201,10 @@ const (
 	// in an atomic manner.
 	ProvisioningClassAtomicScaleUp string = "atomic-scale-up.autoscaling.x-k8s.io"
 )
+
+const (
+	// ProvisioningRequestPodAnnotationKey is a key used to annotate pods consuming provisioning request.
+	ProvisioningRequestPodAnnotationKey = "cluster-autoscaler.kubernetes.io/consume-provisioning-request"
```

Comment:
Hmm, @MaciekPytel we probably want `x-k8s.io` instead of `kubernetes.io` here as well, right? @yaroslava-serdiuk there aren't any pods in the wild using these annotations yet, right?

Reply:
Agreed, it is completely inconsistent with the rest of the names in this API. If there are any pods using it already, we may need to do a deprecation where we support both for a while, but we need to be consistent with those APIs. 

If it's just a cluster autoscaler API, we should put cluster autoscaler in API group and everywhere else. If it's kubernetes API, than it shouldn't use annotation that specifically refers to CA

Reply:
Do you mean we need to have "cluster-autoscaler.autoscaling.x-k8s.io/consume-provisioning-request"?